actionview/lib/action_view/railtie.rb

```diff
@@ -14,6 +14,7 @@ class Railtie < Rails::Engine # :nodoc:
     config.action_view.image_decoding = nil
     config.action_view.apply_stylesheet_media_default = true
     config.action_view.prepend_content_exfiltration_prevention = false
+    config.action_view.parse_rendered_content_in_tests = false
```

Comment:
Not sure if it is worthy guarding this behind a flag. Let's always include the feature. If people define that method, their method will have precedence.